cli/src/logger/sematext.rs

```diff
@@ -0,0 +1,50 @@
+use chrono::Utc;
+use elasticsearch::{http::transport::Transport, Elasticsearch, IndexParts};
+use log::{Level, Metadata, Record};
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use std::env;
+use tokio;
+
+static ES_CLIENT: Lazy<Elasticsearch> = Lazy::new(|| {
+  Elasticsearch::new(Transport::single_node("https://logsene-receiver.sematext.com").unwrap())
+});
+
+#[derive(Deserialize, Serialize, Debug)]
+pub struct Sematext;
+
+impl log::Log for Sematext {
+  fn enabled(&self, metadata: &Metadata) -> bool {
+    metadata.level() <= Level::Info
+  }
+
+  fn log(&self, record: &Record) {
+    let utc_time = Utc::now().format("%FT%T%.3fZ");
+    let env = env::var("ALAN_TECH_ENV").unwrap_or("production".to_string());
+    let cluster = record.target(); // If no target defined default to `anycloud`
+    let token = match env.as_str() {
+      "local" => "f3c3fe7c-9689-470c-98c6-bc60e9b9649d",
+      "staging" => "f3c3fe7c-9689-470c-98c6-bc60e9b9649d",
+      "production" => "f3c3fe7c-9689-470c-98c6-bc60e9b9649d",
+      _ => "",
+    };
```

Comment:
if all environments have the same token and that is another dimension/field in the log itself the we can just set it once? and if so, can we just make it a `const` at the top of the file?